modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-ibm-storage_{context}"]
+= Setting up {ibm-cloud-title} Object Storage
+
+You can set up {ibm-cloud-title} Object Storage on the command line.
+
+.Prerequisites
+
+* You have installed the latest version of the OpenShift CLI. For more information, see "Getting started with the OpenShift CLI" in _CLI tools_.
+
+* You have installed the latest version of the {ibm-cloud-title} Command Line Interface (AWS CLI). For more information, see "Getting started with the {ibm-cloud-title} CLI" in _{ibm-cloud-title} Docs_.
+
+* You have an {ibm-cloud-title} Platform account and ordered an {ibm-cloud-title} Object Storage plan and created an instance of {ibm-cloud-title} Object Storage. For more information, see "Choosing a plan and creating an instance" in _{ibm-cloud-title} Docs_.
+
+.Procedure
+
+. Fetch your credentials from the cluster deployed on {ibm-cloud-title}:
++
+[source,terminal]
+----
+export IBMCLOUD_API_KEY=$(oc get secrets ibmcloud-creds -n kube-system -o json | jq -r '.data.apiKey' | base64 -d)
+----
++
+. Log in to {ibm-cloud-title} by using {ibm-cloud-title} CLI:
++
+[source,terminal]
+----
+$ ibmcloud login -r us-east 
+----
++
+. Install the `cloud-object-storage` and `infrastructure-service` plugins:
++
+[source,terminal]
+----
+$ ibmcloud plugin install cloud-object-storage \
+  && \
+  ibmcloud plugin install infrastructure-service
+----
++
+. Create a resource group:
++
+[source,terminal]
+----
+$ ibmcloud resource group-create tempo-tracing \
+  && \
+  ibmcloud target -g tempo-tracing
+----
++
+. Create a service instance for the object store:
++
+[source,terminal]
+----
+$ ibmcloud resource service-instance-create -d premium-global-deployment \
+  <tempo_bucket> cloud-object-storage standard global \
+  && \
+  export IBM_SERVICE_INSTANCE_ID=$(ibmcloud resource service-instance <tempo_bucket> -o json | jq -r '.[0].crn')
+----
++
+. Create an object store bucket:
++
+[source,terminal]
+----
+$ ibmcloud cos bucket-create --bucket <tempo_bucket> --ibm-service-instance-id "$IBM_SERVICE_INSTANCE_ID"
+----
++
+. Create a service key for connecting to the object store bucket:
++
+[source,terminal]
+----
+$ ibmcloud resource service-key-create <tempo_bucket> Writer --instance-name <tempo_bucket> --parameters '{"HMAC":true}'
```

Comment:
We shouldn't be documenting the complete steps for creating the IBM cloud object store bucket. If we want to document it we should add a note that the steps are for reference and might differ. The only important step is that when creating the service key, it is created with HMAC: true as documented in the step.  

TLDR: we have two options to document this.
*Either document all the steps with a note that these steps are for reference and the steps might change for creation of the IBM cloud object store bucket.
or
* Remove everything before the step ibmcloud resource service-key-create and keep the rest excluding the json_output part.